resources/views/components/dashboards/blocks/company-role-decider.blade.php

```diff
@@ -0,0 +1,100 @@
+@php use \Illuminate\Support\Facades\Auth @endphp
+
+<div>
+    <dl class="pt-11 pb-5 px-6">
+        <div
+            class="py-5 px-4 rounded-lg overflow-hidden relative bg-partner-100 dark:bg-partner-900 shadow-md dark:shadow-md">
+            <dt>
+                <div class="p-3 rounded-md absolute bg-partner-500">
+                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24"
+                         stroke-width="1.5" stroke="white"
+                         aria-hidden="true" class="w-6 h-6">
+                        <path stroke-linecap="round" stroke-linejoin="round"
+                              d="M12 9v3.75m9-.75a9 9 0 1 1-18 0 9 9 0 0 1 18 0Zm-9 3.75h.008v.008H12v-.008Z"/>
+                    </svg>
+                </div>
+            </dt>
+            <div class="ml-16 font-semibold text-md text-gray-700 dark:text-gray-100 overflow-hidden text-ellipsis">
+                <p>You are part of the people joining {{Auth::user()->company->name}} for the "We are in IT together"
+                   conference!<br>
+                   To determine your role within the company during the conference, you will need to do one of the
+                   following:
+                </p>
+                <div class="pt-8 grid grid-cols-1">
+                    <div class="w-full">
+                        If you wish to be a speaker and present on behalf of the company
+                    </div>
+                    @foreach($speakerButtons as $label => $route)
+                        @if ($loop->last && count($speakerButtons) > 1)
+                            <div class="pt-2">
+                                <span>or</span>
+                            </div>
+                        @endif
+                        <div class="pt-2">
+                            <!-- As per current implementation, it's an array it must be a POST request;
+                            refer the backend of the component for more-->
```

Comment:
![image](https://github.com/HZ-HBO-ICT/it-conference/assets/54664131/651d9fba-6147-4d0f-a667-9d0389554180)

See my other comment.